css/properties.json

```diff
@@ -690,8 +691,11 @@
       "-webkit-text-stroke-color"
     ],
     "appliesto": "allElements",
-    "computed": "asSpecified",
-    "order": "uniqueOrder",
+    "computed": [
+      "-webkit-text-stroke-width",
+      "-webkit-text-stroke-color"
+    ],
+    "order": "orderOfAppearance",
```

Comment:
Order of appearance seems wrong, because the color and width can appear in any order. Maybe "canonicalOrder" is better here.

Sebastian